config/checkstyle/checkstyle.xml

```diff
@@ -1,242 +0,0 @@
-<?xml version="1.0"?>
-<!DOCTYPE module PUBLIC
```

Comment:
Keep/bring back this file - the one to delete is .checkstyle .  Other than that I think all the comments have been addressed.